CONTRIBUTING.md

```diff
@@ -1,94 +1,121 @@
 Contributing
 ============
 
-In this document: 
+Thank you for your interest in contributing to the .NET documentation!
+
+In this topic, you'll see the basic process for adding or updating content in our [documentation site](https://docs.microsoft.com/dotnet). For the detailed step-by-step process and instructions, please see the [official guide](https://github.com/Microsoft/Docs/blob/master/readme.md) in the [Microsoft/Docs](https://github.com/Microsoft/Docs) repo.
+
+In this topic, we'll cover: 
 
 * [Adding content](#adding-content)
 * [Process for contributing](#process-for-contributing) 
 * [Guidance checklist](#guidance-checklist)
 * [Building the docs](#building-the-docs)
 * [Contributing to samples](#contributing-to-samples)
+* [Contributor License Agreement](#contributor-license-agreement)
 
 ## Adding content ##
 
-Before adding content, please submit an issue describing your content proposal. For smaller changes or additions, feel free to make a pull request.
+Before adding content, please submit an issue describing your content proposal. For smaller changes or additions, feel free to make a Pull Request (PR) without opening an issue.
+
+The content inside the **docs** folder is organized into sections that are reflected in our Table of Contents (TOC). 
+See where your article fits in the TOC and navigate to the folder associated to that section.
+That folder contains the Markdown files for all articles in the section.
+If necessary, create a new folder to place the files for your content. The main article for that section will be the index.md. 
```

Comment:
It's a little confusing that in the intro to the section we say start with Issue, but here we give guidance as if you're already creating files for a PR.  Maybe preface this paragraph or the latter part of it with "If you get a go-ahead for your proposal ..."